packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.js

```diff
@@ -131,6 +154,71 @@ export default class QuanticSort extends LightningElement {
     this.hasInitializationError = true;
   }
 
+  /**
+   * Sets the error when custom sort options have an invalid configuration.
+   */
+  setSortOptionsConfigurationError() {
+    this.errorMessage = `Custom sort options configuration is invalid.`;
+  }
+
+  /**
+   * Generates the schemas for the validation of the sort options with Bueno.
+   */
+  generateSchemas() {
+    this.headless = getHeadlessBundle(this.engineId);
+    const requiredNonEmptyString = new Bueno.StringValue({
+      required: true,
+      emptyAllowed: false,
+    });
+
+    this.schemas.sortOptionSchema = new Bueno.Schema({
+      label: requiredNonEmptyString,
+      value: requiredNonEmptyString,
+      criterion: new Bueno.RecordValue({
+        options: {
+          required: true,
+        },
+        values: {
+          by: requiredNonEmptyString,
+        },
+      }),
+    });
+
+    this.schemas.sortByDateCriterionSchema = new Bueno.Schema({
+      by: requiredNonEmptyString,
+      order: requiredNonEmptyString,
+    });
+
+    this.schemas.sortByFieldCriterionSchema = new Bueno.Schema({
+      by: requiredNonEmptyString,
+      field: requiredNonEmptyString,
+      order: requiredNonEmptyString,
+    });
+  }
+
+  /**
+   * Validates that the label and values are non-empty strings and that the criterion of the custom sort option is an object which respects the following:
+   * By should be a string
+   * If sorting by field, field must be a string
+   * If sorting by field or date, order must be a string
+   */
```

Comment:
```suggestion
  /**
   * Validates that a sort option respects the required schema of a SortOption.
   */
```

Reply:
If you put the additional info in the comments like you did, we must remember to update the comments if ever the schemas change.